features/utils/world.ts

```diff
@@ -32,7 +34,7 @@ export class OurWorld extends World {
     return this._credentials;
   }
 
-  public set credentials(value: Credential[]) {
+  public set credentials(value: Credentials) {
```

Comment:
It's better to add `init()` method and remove the setter for `credentials`.

We don't expect `credentials` to be changed while testing.
Deleting the setter lets developers know they should not update credentials excepting via `init()`

NOTE: We also want to forbid updating `workingDir`, but now it is updated on `Before("@isolated")`.